components/dashboard/public/image/none-dark.svg

```diff
@@ -0,0 +1 @@
+<svg width="64" height="64" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#a)"><path fill-rule="evenodd" clip-rule="evenodd" d="M50.102 13.898c-9.998-9.997-26.207-9.997-36.204 0s-9.997 26.207 0 36.204 26.206 9.997 36.204 0c9.997-9.997 9.997-26.206 0-36.204ZM36.525 41.051a3.2 3.2 0 0 0 4.526-4.526L27.474 22.95a3.2 3.2 0 1 0-4.525 4.525l13.576 13.577Z" fill="#A8A29E"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h64v64H0z"/></clipPath></defs></svg>
```

Comment:
**Praise**: Thanks for optimizing the SVGs! 🙏

Not super important, but we already have the same icon (rotated) over here:

https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/icons/StatusCanceled.svg

No need to fix this here, but in general I think it would be good to make the location & naming of our icons more consistent.

Reply:
I have moved and renamed SVGs